### ISSUE TITLE
Implement logging of more logits

### ISSUE THREAD

**Issue by dtch1997** [dtch1997/repepo#144]
Currently, our `EvalPrediction` only saves the logit of the token in the completion. We want to additionally save statistics about other logits. Unsure how this could be best implemented. Maybe we add more fields to `TextProbs`.
